projects/router/routes.js

```diff
@@ -9,3 +9,14 @@ FlowRouter.route("/projects/add", {
     BlazeLayout.render("mainLayout", {main: "addProject"});
   }
 });
+
+JsonRoutes.add("get", "/projects/:id", function (req, res, next) {
+  var id = req.params.id;
```

Comment:
Please use a more descriptive variable name here. E.g. `projectId`

This is a good habit to start, as it improves code readability, particularly in complex files.